lib/octokit/configurable.rb

```diff
@@ -113,24 +117,22 @@ def same_options?(opts)
     end
 
     def api_endpoint
-      File.join(@api_endpoint, "")
+      File.join(@api_endpoint, '')
     end
 
     def management_console_endpoint
-      File.join(@management_console_endpoint, "")
+      File.join(@management_console_endpoint, '')
     end
 
     # Base URL for generated web URLs
     #
     # @return [String] Default: https://github.com/
     def web_endpoint
-      File.join(@web_endpoint, "")
+      File.join(@web_endpoint, '')
     end
 
     def login
-      @login ||= begin
```

Comment:
I'm not sure what the significance of the `begin` is here, but Rubocop seems to want to remove it. This might take some research.

Reply:
Looks like this is here for the reasons described [here](https://stackoverflow.com/questions/13279217/are-there-unintended-consequences-of-rubys-begin-end-without-rescue-use). Using brackets will achieve the same thing in a more idiomatic way.